specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "StorageTasksManagementClient",
+    "description": "The Azure Storage Tasks Management API.",
+    "version": "2023-01-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.StorageTasks/operations": {
```

Comment:
>Microsoft.StorageTasks [](http://example.com/codeflow?start=16&length=22)

Is this an externally visible RP? Would any of the resource types be available to external customers? If yes, please review the RP namespace from Naming Consul, if not already done: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices).

Reply:
This was done
https://github.com/Azure/azure-sdk-pr/issues/1205